src/SplitPackages/Program.cs

```diff
@@ -227,7 +227,7 @@ private ProjectJsonFileBuilder CreateBaseJsonFileBuilderForCache(string destinat
 
             var hardcodedDependencies = new[]
             {
-                new PackageInformation(null, "Microsoft.NetCore.App", "1.0.0", new[] { Frameworks.NetCoreApp10 })
+                new PackageInformation(null, "Microsoft.NetCore.App", "1.1.0", new[] { Frameworks.NetCoreApp10 })
```

Comment:
This is used to generate the cache.project.json and final.cache.project.json file. These two files are used to pick correct packages for package cache.

Are these two files used for other purpose? Or is there other unexpected outcome of this tool?

/cc @javiercn